### PR TITLE
update sites.yml

### PR DESCRIPTION
**Pull request by raveling**
## Description

Add newly built Gatsby site to Showcase

### CODE REVIEW

File: docs/sites.yml [hunk 1]
```diff
@@ -9859,6 +9859,17 @@
     - Agency
   built_by: App Design
   built_by_url: "https://appdesign.dev/"
+- title: Clinka
+  url: https://www.clinka.com.au/
+  main_url: https://www.clinka.com.au/
+  source_url:
```
Comment (laurieontech) [inline docs/sites.yml]:
```suggestion
```

Comment (raveling) [reply docs/sites.yml]: Thanks and sorry I didn't catch that myself!

Comment (laurieontech) [reply docs/sites.yml]: Not  a problem!

**Comment by laurieontech** [comment]
Thanks for the update! Looks like the tests are failing because "Environmental" is not a valid category. Feel free to remove that and select another one from the list:
[API, Accessibility, Agency, App, Blog, Business, Community, Conference, Consulting, Data, Design, Design System, Directory, Documentation, Education, Entertainment, Entrepreneurship, Event, Featured, Finance, Food, Freelance, Gallery, Government, Healthcare, Human Resources, JavaScript, Law, Landing Page, Learning, Library, Marketing, Media, Mobile Development, Music, Nonprofit, Open Source, Photography, Podcast, Portfolio, Productivity, Programming, Real Estate, Retail, SEO, Science, Security, Sports, Technology, Travel, User Experience, Video, Web Development, WordPress, eCommerce]

**Comment by laurieontech** [comment]
Thanks for the changes! Looks good.

**Pull request merged**